src/styles/index.scss

```diff
@@ -1,3 +1,46 @@
 body {
   margin: 0;
 }
+
+.card {
+  max-width: 200px;
+
+  background-color: bisque;
+  box-sizing: border-box;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.card__picture {
+  max-width: 160px;
+}
+
+.card__item--model {
+  font-family: Roboto, sans-serif;
+}
+
+.card__item--price {
+  display: flex;
+  justify-content: space-between;
+
+}
```

Comment:
space-between isn't working because you need to define width (100%) for this element (now it's the size of it's elements)